tests/by-util/test_touch.rs

```diff
@@ -1006,3 +1006,15 @@ fn test_obsolete_posix_format_with_year() {
     assert!(at.file_exists("11111111"));
     assert!(!at.file_exists("0101000090"));
 }
+
+#[test]
+fn test_touch_f_option() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let file = "test_f_option.txt";
+
+    ucmd.args(&["-f", file]).succeeds().no_stderr().no_stdout();
```

Comment:
You can simplify it with `no_output()`:
```suggestion
    ucmd.args(&["-f", file]).succeeds().no_output();
```